Bottom: b4647ef3698bedb71f84cdc27550db9457735f24
Top:    e9368154deaa5d3f65c284ca6b98b8c6861bc6fa
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-12 00:25:58 -0700

Refresh of eliminate-virtual-methods-from

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index 1561f4a31c3..1fb3f05fcf9 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -372,6 +372,7 @@ llvm::Expected<long long> As<long long>(llvm::Expected<PythonObject> &&obj);
 template <class T> class TypedPythonObject : public PythonObject {
 public:
   // override to perform implicit type conversions on Reset
+  // This can be eliminated once we drop python 2 support.
   static void Convert(PyRefType &type, PyObject *&py_obj) {}
 
   using PythonObject::Reset;
